.github/actions/setup-dependencies/action.yml

```diff
@@ -97,54 +97,6 @@ runs:
       shell: powershell
       if: runner.os == 'Windows'
 
-    #
```

Comment:
Move the MATLAB setup to a separate action to be able to use it from both CI and MATLAB Analyzer workflows.

Reply:
The problem was how setup-dependencies detect if MATLAB has to be installed:

```
 if: runner.os == 'Linux' && matrix.config == 'matlab'
```

There is no matrix config with the new job.